example.html

```diff
@@ -1,41 +1,38 @@
-<html>
-  <body>
-    <h1>Publish Subscribe example</h1>
-    <button id="publish">Publish sound messages</button>
-    <br><br>
-    <button id="history">Retrieve message history</button><br>
-    <textarea id="history-display" style="width: 100%" rows="15" placeholder="History display..."></textarea>
-  </body>
-
-  <!-- Include the latest Ably Library  -->
-  <script src="https://cdn.ably.io/lib/ably.min.js"></script>
-
-  <!-- Instance the Ably library  -->
-  <script type="text/javascript">
-    alert("You need to add your API key"); /* REMOVE THIS */
-    var realtime = new Ably.Realtime("INSERT-YOUR-API-KEY-HERE"); /* ADD YOUR API KEY HERE */
-
-    /* Get a channel in the "persisted" namespace */
-    var channel = realtime.channels.get("persisted:sounds");
-
-    /* Publish three messages when the publish button is clicked */
-    document.getElementById("publish").addEventListener("click", function() {
-      channel.publish("play", "bark");
-      channel.publish("play", "meow");
-      channel.publish("play", "cluck");
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ably History</title>
+  </head>
+  <body></body>
```

Comment:
`<body></body>`

the `script` needs to be in the `body`

Reply:
Doh - done